tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -238,10 +238,11 @@ def run(self):
                             NS.tendrl_context.integration_id,
                             vol_id=vol_id
                         ).load()
-                        dest = dict(volume.options)
```

Comment:
volume option details are only populated by provisioning node if provisioning node child job fails then volume.option is None so dict(volume.options) will exception so below this we are actually doing "first_sync" is yes because of this exception it won't happen so child job stuck and unmanage also fails.

Reply:
Ok. Good catch of a corner catch. Thanks for fixing this.